### PR TITLE
hardening/326_libthrift_issue_fix_from_pom_xml

### PR DESCRIPTION
**Pull request by pcoello25**
* Fix issue #326 
* Cygnus installed on my computer without any problem. 
I checked my installation path, in `/usr/cygnus/lib` where `libthrift-0.7.0.old` has the .old extension and `libthrift-0.9.1.jar` doesn't appear due to it's included in the .jar file `cygnus-0.11.0-jar-with-dependencies.jar`
* Assignee @frbattid & @vgarciag

### CODE REVIEW

File: doc/installation_and_administration_guide/install_from_sources.md [hunk 1]
```diff
@@ -62,8 +62,6 @@ Apache Flume can be easily installed by downloading its latests version from [fl
     $ mv apache-flume-1.4.0-bin APACHE_FLUME_HOME
     $ mv APACHE_FLUME_HOME/lib/httpclient-4.2.1.jar APACHE_FLUME_HOME/lib/httpclient-4.2.1.jar.old
     $ mv APACHE_FLUME_HOME/lib/httpcore-4.2.1.jar APACHE_FLUME_HOME/lib/httpcore-4.2.1.jar.old
-    $ wget http://repo1.maven.org/maven2/org/apache/thrift/libthrift/0.9.1/libthrift-0.9.1.jar -P APACHE_FLUME_HOME/lib
-    $ mv APACHE_FLUME_HOME/lib/libthrift-0.7.0.jar APACHE_FLUME_HOME/lib/libthrift-0.7.0.jar.old
```
Comment (frbattid) [inline doc/installation_and_administration_guide/install_from_sources.md]: This `mv` command is necessary. Just think this is the manual installation, if we don't rename the old libthrift library, nobody will do that.

**Comment by frbattid** [comment]
CHANGES_NEXT_RELEASE update is missing.

File: doc/installation_and_administration_guide/install_from_sources.md (hunk 1 above)
Comment (pcoello25) [inline doc/installation_and_administration_guide/install_from_sources.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/4374a6b983ae9dce1296548d9e78ec16a595cf9b

**Comment by pcoello25** [comment]
CHANGES_NEXT_RELEASE updated in https://github.com/telefonicaid/fiware-cygnus/commit/4374a6b983ae9dce1296548d9e78ec16a595cf9b

**Comment by frbattid** [comment]
LGTM, passing the ball to @vgarciag

**Comment by vgarciag** [comment]
LGTM

**Pull request merged**